scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1758,6 +1769,14 @@ def test_concatenate():
     result = Rotation.concatenate(split)
     assert_equal(rotation.as_quat(), result.as_quat())
 
+    # Test Rotation input for multiple rotations
+    result = Rotation.concatenate(rotation)
```

Comment:
Can you also test that the result is a copy?

Reply:
Sure thing, done